### ISSUE TITLE
[BUG] Misleading/insufficient documentation for Tabs

### ISSUE THREAD

**Issue by kodeaben** [kirbydesign/designsystem#2761]
- [x] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the bug
 

The cookbook showcase for Tabs refers to a `TabsPage`, which doesn't exist.
The loadChildren example in the route file assigns a string, instead of a function. This won't compile: 
``` Typescript
...
 path: 'dashboard',
        children: [
          {
            path: '',
            loadChildren: '../dashboard/dashboard.module#DashboardModule' <-- here
          }
        ]
      },
...
```

## Please complete the following information:
- OS: [e.g. Windows, MacOS, iOS]
- Browser [e.g. Chrome, Safari]
- Browser version [e.g. 22]

## Are there any additional context?

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Verification
To make sure the bug is not intended behaviour; it should be verified by a member of team Kirby before moving on to implementation. 

- [x] Make sure the `NOT verified` label has been removed by a member of [team Kirby](https://github.com/kirbydesign/designsystem/blob/main/.github/SUPPORT.md#team-kirby); do not proceed until this is done

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Create a test that reproduces the bug following guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)". 
- [ ] Write code to fix the bug

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request.

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
